From 095face02297128890801f9fd18d4acc7675713c Mon Sep 17 00:00:00 2001
From: Christine Gharzuzi <chrisg@marvell.com>
Date: Tue, 29 May 2018 16:16:03 +0300
Subject: [PATCH 0638/1051] dts: i2c: a8kp: enable CP0 I2C0 for DB boards

I2C0 in CP0 is enabled for all a8kplus existing
configurations in u-boot.

This patch enables I2C0 in CP0 for a8kplus kernel
device-trees

Change-Id: Ib08736527192d80682c572f60923db988352fe5e
Signed-off-by: Christine Gharzuzi <chrisg@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-8082-db.dts | 4 ++++
 arch/arm64/boot/dts/marvell/armada-8084-db.dts | 4 ++++
 arch/arm64/boot/dts/marvell/armada-8162-db.dts | 4 ++++
 arch/arm64/boot/dts/marvell/armada-8164-db.dts | 4 ++++
 4 files changed, 16 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-8082-db.dts b/arch/arm64/boot/dts/marvell/armada-8082-db.dts
index 3ad5a889bd7d..137ba52df043 100644
--- a/arch/arm64/boot/dts/marvell/armada-8082-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8082-db.dts
@@ -130,6 +130,10 @@
 	};
 };
 
+&cp0_i2c0 {
+	status = "okay";
+};
+
 &cp0_ethernet {
 	status = "okay";
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-8084-db.dts b/arch/arm64/boot/dts/marvell/armada-8084-db.dts
index 9628bba0fd84..f9e665d107de 100644
--- a/arch/arm64/boot/dts/marvell/armada-8084-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8084-db.dts
@@ -140,6 +140,10 @@
 	};
 };
 
+&cp0_i2c0 {
+	status = "okay";
+};
+
 &cp0_ethernet {
 	status = "okay";
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-8162-db.dts b/arch/arm64/boot/dts/marvell/armada-8162-db.dts
index 8d956ac5e28e..82364d49f9cf 100644
--- a/arch/arm64/boot/dts/marvell/armada-8162-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8162-db.dts
@@ -106,6 +106,10 @@
 	phy-names = "pcie-phy0", "pcie-phy1", "pcie-phy2", "pcie-phy3";
 };
 
+&cp0_i2c0 {
+	status = "okay";
+};
+
 &cp0_pcie1 {
 	msi-parent = <&ap0_gic_its>;
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-8164-db.dts b/arch/arm64/boot/dts/marvell/armada-8164-db.dts
index 6aa824c34e67..483133e7bfd4 100644
--- a/arch/arm64/boot/dts/marvell/armada-8164-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8164-db.dts
@@ -92,6 +92,10 @@
 	non-removable;
 };
 
+&cp0_i2c0 {
+	status = "okay";
+};
+
 &cp0_sata0 {
 	status = "okay";
 	sata-port@1 {
-- 
2.17.1

